pkgs/development/python-modules/sqlalchemy-utils/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, six, sqlalchemy
+, pytz, mock, pytest, flexmock, python-dateutil }:
+
+buildPythonPackage rec {
+  pname = "sqlalchemy-utils";
+  version = "0.33.6";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "SQLAlchemy-Utils";
+    sha256 = "45ab41c90bfb8dd676e83179be3088b3f2d64b613e3b590187163dd941c22d4c";
+  };
+
+  propagatedBuildInputs = [
+    six
+    sqlalchemy
+  ];
+
+  doCheck = false;
```

Comment:
Why?

Reply:
The tests are accessing the network (localhost). That's why they're failing. And if that weren't prohibited by the sandbox, there still would be no database to access.